install_config/web_console_customization.adoc

```diff
@@ -855,6 +855,32 @@ xref:../install_config/configuring_authentication.adoc#OpenID[OpenID] identity
 providers, which require visiting an external URL to destroy single sign-on
 sessions.
 
+[[changing-the-membership-whitelist]]
+== Changing the Membership Whitelist
+
+The default whitelist in the membership page will show a certain subset of Cluster Roles
```

Comment:
The default whitelist in the membership page shows a subset of cluster roles, such as `admin`, basic-user`, `edit`, and so on. It also shows custom roles defined within a project.